### ISSUE TITLE
learning rate is no more lr but learnig_rate

### ISSUE THREAD

**Issue by ezerbib** [aladdinpersson/Machine-Learning-Collection#178]
https://github.com/aladdinpersson/Machine-Learning-Collection/blob/558557c7989f0b10fee6e8d8f953d7269ae43d4f/ML/TensorFlow/Basics/tutorial3-neuralnetwork.py#L42

should be replaced with
optimizer=keras.optimizers.Adam(learning_rate=0.001),

**Comment by Shashank1202** [comment]
@ezerbib Yes, after going through the documentation and well a code, I think it should be learning_rate over lr, So I have solved this issue, please look into my PR #178
